packages/react-rich-text-lexical-renderer/README.md

```diff
@@ -96,12 +101,73 @@ const myTheme = {
                     id: "custom_heading1",
                     name: "Custom Heading 1",
                     tag: "h1",
-                    styles: { ...headings, fontWeight: "bold", fontSize: 48 }
+                    styles: {...headings, fontWeight: "bold", fontSize: 48}
                 }]
         }
     }
 }
 
 // Mount the component
-<RichTextLexicalRenderer value={content}  theme={myTheme} nodes={[MyCustomNode]} />;
+<RichTextLexicalRenderer value={content} theme={myTheme} nodes={[MyCustomNode]}/>;
+```
+
+## Resolve the mismatch of the versions in the React v18 application
+
+When you try to use `RichTextLexicalRenderer` component in React `v18` application you will see this error on the
+screen:
+
+![React application error for mismatch of the React versions](./images/react-renderer-versisons-conflict-error.png)
+
+This is because our `@webiny/react-rich-text-lexical-renderer` package and the React application have
+different versions of React. Our rich text renderer component is using `v17.0.2`, and the React application is
+using `v18.x.x`.
+
+> You can check the React version dependencies by running following command:
+> - `yarn why react` for `yarn` users.
+> - `npm ls react` for `npm` users.
+
+To resolve this problem, we need to define the versions of the `@webiny/react-rich-text-lexical-renderer`
+package, and it's dependencies to `v18.x.x`.
+
+### Instructions for `yarn` users
+
+To define the `v18.x.x` of React to all package dependencies that use the old version of React, we need to
```

Comment:
To force `yarn` to resolve dependencies across the project to the exact versions we're looking for, use the `resolutions` field in the root `package.json` file.